webapp/components/description/description.js

```diff
@@ -1,148 +1,148 @@
 import './description.less'
 
-import React, { Component } from 'react'
-import ReactDOM from 'react-dom'
+import React, { useRef, useEffect } from 'react'
 import PropTypes from 'prop-types'
-import { connect } from 'react-redux'
+import { useSelector, useDispatch } from 'react-redux'
 import * as R from 'ramda'
 
 import Icon from '@webapp/components/icon'
 import Tooltip from '@webapp/components/tooltip'
 import ckEditorConfig from '@webapp/components/ckEditor/ckEditorConfig'
 
-import * as AppState from '@webapp/app/appState'
+import useI18n from '@webapp/components/hooks/useI18n'
+import useCountryIso from '@webapp/components/hooks/useCountryIso'
+import useUserInfo from '@webapp/components/hooks/useUserInfo'
+import * as DescriptionState from '@webapp/components/description/descriptionState'
 
 import { saveDescriptions, fetchDescriptions, openEditor, closeEditor } from './actions'
 
-class Description extends Component {
+const Description = props => {
```

Comment:
- I would extract the two components into separate files
- I would use propTypes and defaultProps